go/integration_test/processor/create_test.go

```diff
@@ -633,7 +634,71 @@ func TestProcessor_CodeStartingWith0xEFCanNotBeCreated(t *testing.T) {
 					if result.GasUsed != sufficientGas {
 						t.Errorf("execution failed but gas was not fully used, used %d", result.GasUsed)
 					}
+				}
+			})
+		}
+	}
+}
+
+func TestProcessor_CodeSizeIsLimited(t *testing.T) {
+	tests := map[string]struct {
+		length  int
+		success bool
+	}{
+		"threshold": {
+			length:  floria.MaxCodeSize,
```

Comment:
Sorry, I did not realize that the constant was in the `floria` package and this test is in the `integration_test` package. In this case, of cause, you have been right in re-defining the constant here.

Reply:
Changed it back to a local constant.